flutter/lib/src/utils/multi_view/html_multi_view_helper.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:ui';
+import 'multi_view_helper.dart';
+
+MultiViewHelper multiViewHelper() => HtmlMultiViewHelper();
+
+class HtmlMultiViewHelper implements MultiViewHelper {
+  @override
+  bool isMultiViewEnabled() {
```

Comment:
why do we need the split into different platforms? io, web, html

Reply:
you are right, with my previous attempt of reading js variables, I needed to split things up, because this would obviously only work on `web`. `PlatformDispatcher.instance.implicitView` is not bound to any particular platform.